CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to
 
 ## [Unreleased]
 
+- **Breaking**: Remove `dbt-core` from dependencies. Since it is not mandatory
+  for `dbt-score` to execute `dbt`, remove the dependency.
```

Comment:
```suggestion
- **Breaking**: The rule `public_model_has_example_sql` has been renamed
  `has_example_sql` and applies by default to all models.
- **Breaking**: Remove `dbt-core` from dependencies. Since it is not mandatory
  for `dbt-score` to execute `dbt`, remove the dependency.
```
While there, I did forget to document this change 😇

Reply:
Ah good one :+1: